api/nntrainer-api-common.h

```diff
@@ -95,13 +95,16 @@ typedef enum {
  * number of samples are set as a property for this dataset. In this case, last
  * is only used for verification purposes. If total number of samples for the
  * dataset is unknown, this function will be called in sequence.
+ * @note If @a last false when not modified by the user. @a last has to be set
```

Comment:
If @a last false when not modified by the user. -> it sounds like `last` variable will be defaulted to false by the caller and user need not set it if not needed. Is this intentional?

Reply:
Yes, it is current behavior and I think it's fine to behave like this.

Reply:
The described behavior works, but IMO, its best for the callee to always set this variable, and not assume about the default value set by the caller.

Reply:
Let's delete this line although this behavior will exist to not break old codes.